Resources/public/inlineDeleteCheckboxes.js

```diff
@@ -0,0 +1,39 @@
+/*
+
+ This file is part of the Sonata package.
+
+ (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+ */
+
+var handleInlineDeleteCheckboxes = function(){
```

Comment:
Please run this throught jslint or jshint or something, I'm pretty sure there should be a space between `)` and `{`